docs/specification/current.md

```diff
@@ -447,6 +447,7 @@ Each **Card** is described by the following attributes.
 
 Field | Optionality | Type | Description
 ----- | ----- | ----- | --------
+`uuid` | OPTIONAL | *string* | Unique identifier of the card, used for auditing and logging cards and to provide feedback.
```

Comment:
It's unclear to me how a uuid provides feedback, consider striking 'and to provide feedback'

Reply:
Brett, the newly added `card.uuid` is primarily needed for the cds client to identify which card it's talking about when it calls the cds service's feedback endpoint. I've tried to clarify that because, to your point, a uuid doesn't provide feedback. :)

If you still think 'and to provide feedback should be scratched, let me know and I'll do so.

Reply:
Brett,

The primary purpose of card.uuid is to identify which card the cds client is talking about when it subsequently calls a cds service's feedback endpoint. I attempted to clarify this, let me know if you still think this phrase should be removed.

Isaac